### PR TITLE
Add instructions for i-PI

### PR DESCRIPTION
**Pull request by tuoping**

### CODE REVIEW

**Review by njzjz — APPROVED** [review]

File: doc/getting-started.md [hunk 1]
```diff
@@ -313,8 +313,9 @@ kspace_modify	gewald 0.45
 Please notice that the DeePMD does nothing to the direct space part of the electrostatic interaction, because this part is assumed to be fitted in the DeePMD model (the direct space cut-off is thus the cut-off of the DeePMD model). The splitting parameter `gewald` is modified by the `kspace_modify` command.
 
 ### Run path-integral MD with i-PI
-The i-PI works in a client-server model. The i-PI provides the server for integrating the replica positions of atoms, while the DeePMD-kit provides a client named `dp_ipi` that computes the interactions (including energy, force and virial). The server and client communicates via the Unix domain socket or the Internet socket. The client can be started by
+The i-PI works in a client-server model. The i-PI provides the server for integrating the replica positions of atoms, while the DeePMD-kit provides a client named `dp_ipi` that computes the interactions (including energy, force and virial). The server and client communicates via the Unix domain socket or the Internet socket. Installation instructions of i-PI can be found [here](install.md#install-i-pi). The client can be started by
 ```bash
+$ i-pi input.xml
```
Comment (amcadmus) [inline doc/getting-started.md:318]: Shouldn't ipi be put background?

File: doc/getting-started.md [hunk 2]
```diff
@@ -313,8 +313,9 @@ kspace_modify	gewald 0.45
...
 $ dp_ipi water.json
```
Comment (njzjz) [inline doc/getting-started.md:319]: I suggest to remove `$` as it's useless.
```suggestion
i-pi input.xml &
dp_ipi water.json
```

File: doc/getting-started.md (hunk 1 above)
Comment (tuoping) [reply doc/getting-started.md:318]: I'm not sure. I need to confirm it. Sorry.

Comment (tuoping) [reply doc/getting-started.md]: Yes, ipi should be put background. And the error occured on EHPC seems to happen whenever I invoke lammps, which is not a problem with i-PI. Sorry about this. I promise to be more careful next time.

**Review by jameswind — APPROVED** [review]

**Pull request merged**